### PR TITLE
Update main.py

### PR DESCRIPTION
**Pull request by melissatrann**
Set segment implementation... to change colors of neopixels separated in segments

### CODE REVIEW

**Comment by jly02** [comment]
Could you really quickly fix some of the spacing (specifically on lines 49/50). Also, your comments look good, but if you could use the style I gave in the README, that would let us stay consistent with our documentation. You can commit the changes without making a new PR, and once that's done I'll merge it. Thanks!

**Comment by melissatrann** [comment]
> Could you really quickly fix some of the spacing (specifically on lines 49/50). Also, your comments look good, but if you could use the style I gave in the README, that would let us stay consistent with our documentation. Thanks!

Okie

File: main.py [hunk 1]
```diff
@@ -25,14 +25,54 @@
...
+    end (int): The ending index of the segment (inclusive).
+    
+    Example:
+    To define the first half of a 16-pixel strip:
+    define_segment("left_half", 0, 7)
+    To define the second half of a 16-pixel strip:
+    define_segment("right_half", 8, 15)
+    """
+    if 0 <= start <= end < NUM_PIXELS:
+        segments[name] = (start, end)
```
Comment (jly02) [inline main.py:49]: an empty line after this would be good

File: main.py [hunk 2]
```diff
@@ -25,14 +25,54 @@
...
 # Set up distance sensor.
 sensor = machine.ADC(SENSOR_PIN)
+segments = {}
+
+def define_segment(name, start, end):
+    """
+    Defines a segment of the neopixel light
+    
+    Parameters:
+    name (str): The name of the segment.
```
Comment (jly02) [inline main.py]: these params (and the others) should be in the form `:param name: the name of the segment`

Comment (melissatrann) [reply main.py]: lmk if I need to update anything else

**Comment by jly02** [comment]
LGTM

**Comment by melissatrann** [comment]
> LGTM

Hooray 😊😁😆

**Pull request merged**